src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildADesktopExe.cs

```diff
@@ -850,5 +852,154 @@ public void It_allows_TargetFrameworkVersion_to_be_capitalized()
                 .Should()
                 .Pass();
         }
+
+        [WindowsOnlyTheory]
+        [InlineData("true", "true")]
+        [InlineData("true", "false")]
+        [InlineData("false", "true")]
+        [InlineData("false", "false")]
+        public void It_places_package_pdb_and_xml_files_in_output_directory(string enableCopyDebugSymbolFilesFromPackages, string enableDocumentationFilesFromPackages)
```

Comment:
I think it would be better to change these parameters to `bool` values.

Reply:
I think we can leave it as strings since they represent a value assigned to a property and not a value to be used in a logic condition.

I do see that most of the unit-tests are using true instead of "true", they use this values in a logic condition.